glam/api/models.py

```diff
@@ -202,3 +202,19 @@ class Meta:
                 fields=["app_id", "channel", "version", "ping_type", "build_id", "os"],
             )
         ]
+
+
+class FirefoxBuildRevisions(models.Model):
```

Comment:
this class autogenerates the `FirefoxBuildRevisions` table in `0017_add_shas_table.py ` when we run `./manage.py migrate`?

Reply:
This Django model will be used to create a migration file when we run `./manage.py makemigrations`, which I did already and the resulting file is `glam/api/migrations/0017_add_shas_table.py`. Then when we run `./manage.py migrate` it reads this file and creates the database table (or whatever migration action is listed there).